sql/mob_pools.sql

```diff
@@ -2186,7 +2186,7 @@ INSERT INTO `mob_pools` VALUES (2130,'Jailer_of_Faith','Jailer_of_Faith',109,0x0
 INSERT INTO `mob_pools` VALUES (2131,'Jailer_of_Fortitude','Jailer_of_Fortitude',124,0x0600910400000000000000000000000000000000,7,7,3,220,125,0,1,1,0,2,16511,0,0,1157,0,0,4,0,0,124,124);
 INSERT INTO `mob_pools` VALUES (2132,'Jailer_of_Hope','Jailer_of_Hope',194,0x0000820400000000000000000000000000000000,1,4,7,280,125,0,1,1,0,2,9,0,0,2183,5,0,0,24,0,4022,194);
 INSERT INTO `mob_pools` VALUES (2133,'Jailer_of_Justice','Jailer_of_Justice',269,0x0000110200000000000000000000000000000000,9,9,7,200,125,0,1,1,1,2,9,0,0,0,0,0,0,0,0,269,269);
-INSERT INTO `mob_pools` VALUES (2134,'Jailer_of_Love','Jailer_of_Love',271,0x00007C0400000000000000000000000000000000,15,15,7,220,125,0,1,1,0,2,0,0,0,0,1,0,0,0,0,271,271);
+INSERT INTO `mob_pools` VALUES (2134,'Jailer_of_Love','Jailer_of_Love',271,0x00007C0400000000000000000000000000000000,4,15,7,220,125,0,1,1,0,2,0,0,0,0,1,0,0,0,0,271,271);
```

Comment:
Did Jailer of Love need its model size increased? I thought it looked small on your stream today.